src/deploy/functions/tasks.ts

```diff
@@ -120,10 +120,13 @@ export function createFunctionTask(
       operationResourceName: op.name,
       onPoll,
     });
-    if (!backend.isEventTrigger(fn.trigger) && fn.invoker) {
+    if (!fn.invoker) {
```

Comment:
1. Only for https triggers. We will possibly start supporting invoker for other triggers in v2 (you can set EventArc's invoker), but we would never default to public
2. I'm nervous about modifying the fn itself. It might be better to just copy the list out